### ISSUE TITLE
skips xml with no classes or 'object'

### ISSUE THREAD

**Issue by bijonguha** [rihito-bannai/convert_pascal_voc_xml_to_yolo#1]
Wherever xml files with no objects are there, it does not create empty text files

**Comment by rihito-bannai** [comment]
Thank you for your report. Is it a problem? Why do you need empty text files?

**Comment by bijonguha** [comment]
Yes, because when there are empty images where no objects are the, you still need to create empty text files corresponding to those images in dataset.

**Comment by rihito-bannai** [comment]
YOLOv5 skips images without text files, so there is no problem. Don't other YOLOs work the same way? @bijonguha

**Comment by bijonguha** [comment]
Nope, YOLOv5 code is written in a very matured way and it takes care care of many issues like the one mentioned above, it would also install missing packages instead of throwing error too, that's the kind of mature code it is. But not all the implementation of yolo are like that.

**Comment by rihito-bannai** [comment]
@bijonguha OK, thank you very much, I understand. I will solve this issue.
